data-raw/process_data_sets_maggie.R

```diff
@@ -0,0 +1,266 @@
+    startdate = as.Date(startdate, format = "%m / %d / %Y"),
+    enddate = as.Date(enddate, format = "%m / %d / %Y"),
+    pollster = as.factor(pollster),
+    grade = as.factor(grade),
+    population = as.factor(population),
+    url = as.factor(url),
+    createddate = as.Date(createddate, format = "%m / %d / %Y")
+  ) %>%
+  rename(
+    model_date = modeldate,
+    start_date = startdate,
+    end_date = enddate,
+    sample_size = samplesize,
+    created_date = createddate
+  )
+#TODO: unsure how to deal with timestamp-- split into date and time or leave as is?
+#TODO: can I delete "president" and "modeldate" since each only has one value in this dataset (trump and Jan 5)? if so:
+trump_approval_poll <- trump_approval_poll[,-c(1,3)]
+devtools::use_data(trump_approval_poll, overwrite = TRUE)
+
+trump_approval_trend <- read_csv("data-raw/trump-approval-ratings/approval_topline.csv")
+trump_approval_trend <- trump_approval_trend %>%
+  mutate(
+    president = as.factor(president),
+    subgroup = as.factor(subgroup),
+    modeldate = as.Date(modeldate, format = "%m / %d / %Y")
+  ) %>%
+  rename(
+    approve_high = approve_hi,
+    approve_low = approve_lo,
+    disapprove_high = disapprove_hi,
+    disapprove_low = disapprove_lo
+  )
+trump_approval_poll <- trump_approval_poll[,-c(1,3)]
+devtools::use_data(trump_approval_trend, overwrite = TRUE)
+#same comments as above about timestamp, president, model date
+
+# trump-world-trust---------------------------------------------------------------------
+trumpworld_issue_1 <- read_csv("data-raw/trump-world-trust/TRUMPWORLD-issue-1.csv")
+trumpworld_issue_1 <- clean_names(trumpworld_issue_1)
+trumpworld_issue_1$country <- as.factor(trumpworld_issue_1$country)
+trumpworld_issue_1$issue <- 1
+
+trumpworld_issue_2 <- read_csv("data-raw/trump-world-trust/TRUMPWORLD-issue-2.csv")
+trumpworld_issue_2 <- clean_names(trumpworld_issue_2)
+trumpworld_issue_2$country <- as.factor(trumpworld_issue_2$country)
+trumpworld_issue_2$issue <- 2
+
+trumpworld_issue_3 <- read_csv("data-raw/trump-world-trust/TRUMPWORLD-issue-3.csv")
+trumpworld_issue_3 <- clean_names(trumpworld_issue_3)
+trumpworld_issue_3$country <- as.factor(trumpworld_issue_3$country)
+trumpworld_issue_3$issue <- 3
+
+trumpworld_issue_4 <- read_csv("data-raw/trump-world-trust/TRUMPWORLD-issue-4.csv")
+trumpworld_issue_4 <- clean_names(trumpworld_issue_4)
+trumpworld_issue_4$country <- as.factor(trumpworld_issue_4$country)
+trumpworld_issue_4$issue <- 4
+
+trumpworld_issue_5 <- read_csv("data-raw/trump-world-trust/TRUMPWORLD-issue-5.csv")
+trumpworld_issue_5 <- clean_names(trumpworld_issue_5)
+trumpworld_issue_5$country <- as.factor(trumpworld_issue_5$country)
+trumpworld_issue_5$issue <- 5
+
+trumpworld_combined<- rbind(trumpworld_issue_1, trumpworld_issue_2, trumpworld_issue_3, trumpworld_issue_4, trumpworld_issue_5)
+#TODO: would it make sense to merge these datasets like this? If so, can delete devtools for individual issues
```

Comment:
Yes, that is tidy data principle number 3: similar observational units should be in the same table. Call this `trumpworld_issues` instead and use `dplyr::bind_rows()`. While the diff between `rbind` and `bind_rows` materially is little, for larger and larger datasets I believe the dplyr code is faster.